docs/conf.py

```diff
@@ -61,7 +61,7 @@
 #
 
 # The full version, including alpha/beta/rc tags.
-release = '0.10.0 alpha 0'
+release = '0.10.0a0'
```

Comment:
Should we stick to semantic versioning? https://semver.org/
Would mean to put a dash between the patch and the appended `a0`, wouldn't it?

Reply:
I stumbled upon the same thing.

I used it as it is quite common in the python world, https://www.python.org/dev/peps/pep-0440/#pre-releases

But indeed does not comply with semver.

`0.10.0-alpha.1` seems to be semver compliant and to python it seems to be the same as `0.10.0a1`

```
>>> packaging.version.parse("1.2.3-alpha.1")
<Version('1.2.3a1')>

```

Reply:
Changed it for our second alpha release to a new versioning scheme :)

Reply:
> I used it as it is quite common in the python world, https://www.python.org/dev/peps/pep-0440/#pre-releases
> 
> But indeed does not comply with semver.
> 
> `0.10.0-alpha.1` seems to be semver compliant and to python it seems to be the same as `0.10.0a1`
> 
> ```
> >>> packaging.version.parse("1.2.3-alpha.1")
> <Version('1.2.3a1')>
> ```

Interesting! Thanks for trying!